src/kartoffel/kartoffel.intergation.ts

```diff
@@ -24,12 +24,11 @@ export class Kartoffel {
      * @param id - the user ID (return the id that specified in the request)
      * @param dest? - optional param that identify the external destination, if not mentioned look in non-external network
      */
-    async getByID(id: string, dest?: string): Promise<IUser> {
+    async getByID(id: string, dest?: string): Promise<any> {
         let res: AxiosResponse;
         try {
-            const query: string = dest && dest === (EXTERNAL_DESTS.CTS as any as string) ? kartoffelCTSQueryGet : '';
```

Comment:
Why did you remove the support in another destination? When it's CTS dest you'll need to get a user by unique id

Reply:
Added back.